float32 tilt       # Head tilt target angle in radians. 
float32 pan        #  Head pan target angle in radians. 

bool reachable   #  Whether the head motion request points on a reachable position. 
bool moving   #  Whether the head is currently in motion or not. 





